specification/purview/data-plane/Azure.Analytics.Purview.Scanning/preview/2022-07-01-preview/scanningService.json

```
@@ -0,0 +1,9155 @@
+        ],
+        "description": "Enables interactive querying",
+        "operationId": "IntegrationRuntimes_EnableInteractiveQuery",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "integrationRuntimeName",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "schema": {
+              "$ref": "#/definitions/EnableInteractiveQueryForIntegrationRuntimeRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/IActionResult"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_EnableInteractiveQuery": {
+            "$ref": "./examples/IntegrationRuntimes_EnableInteractiveQuery.json"
+          }
+        }
+      }
+    },
+    "/managedvirtualnetworks/{managedVNetName}": {
+      "get": {
+        "tags": [
+          "ManagedVirtualNetworks"
+        ],
+        "description": "Gets a ManagedVirtualNetwork",
+        "operationId": "ManagedVirtualNetworks_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "managedVNetName",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "default"
+            ],
+            "x-ms-enum": {
+              "name": "ManagedVirtualNetworkName",
+              "modelAsString": true
+            }
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ManagedVirtualNetwork"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ManagedVirtualNetworks_Get": {
+            "$ref": "./examples/ManagedVirtualNetworks_Get.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ManagedVirtualNetworks"
+        ],
+        "description": "Creates or Updates a ManagedVirtualNetwork",
+        "operationId": "ManagedVirtualNetworks_CreateOrUpdate",
```

A put operation is a create or replace:
```suggestion
        "description": "Creates or Replaces a ManagedVirtualNetwork",
        "operationId": "ManagedVirtualNetworks_CreateOrReplace",
```

---

changed